opencti-platform/opencti-front/src/utils/Security.tsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react';
+import React, { FunctionComponent, useContext } from 'react';
 import { filter, includes, map } from 'ramda';
 
-export const UserContext = React.createContext({});
+interface Capability {
+  id: string;
+  name: string;
+}
+
+export interface Me {
```

Comment:
Couldn't we use the MeUser type from relay ?

Reply:
Yep, type change to use the relay definition